docs/reference/fall-detection.md

```diff
@@ -0,0 +1,113 @@
+Returns a dictionary containing stats regarding evaluation.
+
+Parameters:
+
+- **dataset**: *object*\
+  Object that holds the evaluation dataset.
+  Can be of type `ExternalDataset` or a custom dataset inheriting from `DatasetIterator`.
+- **verbose**: *bool, default=True*\
+  If set to True, enables the maximum verbosity.
+
+#### `FallDetectorLearner.infer`
+```python
+FallDetectorLearner.infer(img)
+```
+
+This method is used to perform fall detection on an image.
+Returns a list of tuples, one for every person detected, that each contains an `engine.target.Category`, a list of
+three keypoints that define two lines that are used to determine if the person has fallen, and the complete poses that
+were detected.
+It returns an empty list if no pose detections were made.
+
+The `engine.target.Category` is `1` if person has fallen, `-1` if person is standing and `0` if a person is detected, but
+the algorithm is unable to detect if person is standing or fallen.
+
+Parameters:
+
+- **img**: *object*\
+  Object of type engine.data.Image.
+
+#### `FallDetectorLearner.download`
+```python
+FallDetectorLearner.download(self, path, mode, verbose, url)
+```
+
+Download utility for downloading fall detection test images and annotations.
+
+Parameters:
+
+- **path**: *str, default=None*\
+  Local path to save the files, defaults to '.' if None.
+- **mode**: *str, default="test_data"*\
+  What files to download, can only be "test_data"
+- **verbose**: *bool, default=False*\
+  Whether to print messages in the console.
+- **url**: *str, default=OpenDR FTP URL*\
+  URL of the FTP server.
+
+
+#### Examples
+
+* **Inference and result drawing example on a test image using OpenCV.**
+  ```python
+  import cv2
+  from opendr.engine.data import Image
+  from opendr.perception.fall_detection import FallDetectorLearner
+  from opendr.perception.pose_estimation import LightweightOpenPoseLearner
+  from opendr.perception.pose_estimation import draw, get_bbox
+  
+  pose_estimator = LightweightOpenPoseLearner(device="cuda", mobilenet_use_stride=False)
+  pose_estimator.download(verbose=True)  # Download the default pretrained mobilenet model
+  pose_estimator.load("openpose_default")
+  
+  fall_detector = FallDetectorLearner(pose_estimator)
+  
+  # Download a sample dataset
+  fall_detector.download(verbose=True)
+  
+  img = Image.open("test_images/fallen.png")
+  detections = fall_detector.infer(img)
+  fallen = detections[0][0].data  # Get fallen int from first detection
+  pose = detections[0][2]  # Get pose from first detection
+  img = img.opencv()
+  draw(img, pose)  # Draw the detected pose
+  
+  if fallen == 1:
+      x, y, w, h = get_bbox(pose)
+      cv2.rectangle(img, (x, y), (x + w, y + h), (0, 0, 255), 2)
+      cv2.putText(img, "Detected fallen person", (5, 12), cv2.FONT_HERSHEY_SIMPLEX, 0.5, (0, 0, 255), 1, cv2.LINE_AA)
+  cv2.imshow('Result', img)
+  cv2.waitKey(0)
+  ```
+
+#### Notes
+
```

Comment:
I left this on purpose actually, because I've been meaning to ask your opinion whether you feel there is anything that needs further explaining in the docs. I will remove this for now, thanks!